### PR TITLE
updateCounter method (bug correction): append counter only once

### PR DESCRIPTION
**Pull request by RunnX**
There was a bug in the updateCounter method that added a counter element even if the widget already had a counter element.  This would result in two counters showing up for one social widet (Facebook 0 0).

### CODE REVIEW

File: src/styles/flat/font/social-likes.styl [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
...xkGjPAwL0LA2f/4BEXp9JID0aC/Nd6WmpRM7t9RVTbdVqkGNaDpvIPZ1W47gxY2SaBfssPoOnyC9D+oXS9l4Hq6mnds5m9+PZzvZrMrn1v1qyLQB5LWU8fxXZS2oDo+q00fFZUsogp6Q/y5k4qkSjEWcLqSflB07xFt4PHzwzoHduB4MgCcOvGiFZHNfl6UU3v8XMnGDVvJyow36j7clf1LHJkPSjrFwFpGfTfLHwe5f2k87eYc/YjoNtZ3atbMPAH/0E4fxQDv0E+dYMvHwfOmbpZwEDpY/wQ3rd8v2KiNLK2vINZ8RzuBZIRRy9ivd33bp3ebPCBNDff4+upZN5e3D2VYqaLXIL6WuL5rhO1B98x3wPFvkBfjgJKCAvem3/qh67shwwUVA18Hv9gF8vJ1Hni8PKFgEtqKPPERsO2twSFbRSbzyCgYmXq+kGsG4RiUOgSyzTFotcRTYRrg9hraUxubPs3xCNJfoM7eA59ccOBqv4NuigmqpBSw8sdDbmH+/aLnRJ02AzCENZ3NxJ4ut0Y7xaCa8gwHg+VHosKUv3O+qIzQ8wRenf6YS7Mm0u9l562AlyG8Lk8ja9aXfgK6Wb1nMxc0NgBKLya6B6MhhcAXb6KJZuGmBKpL7synHkMSqQacAESYQkU4At304t0hqs0k1c4ZYzwNmJYOqnm48bpV2mCLG4uPMmIx6lFRaCtYrpsvIjPBjMhmH9g2SG96zmIEdof6yRPYvCPbcWg2pt1m+tVULZkCX7pqJhahny+1VRSGBjVnwCN9jDzwWo9+KZ74UPqcwQ3Zb4Ovk+tX6Rq+G+Kjw+vAT7V1vPdo+5ZHnYK6QqoQjqwGgsUYe6ONxVHS2p+K8PEJSYZrmE+FJdesbYttjcZqvnyW3GKhryb2bl2Db13NJrKA5gbweviWRpkz5P07SeYXHDTUDU8gzbXglbVxGdtO59dAhVUikZk/om0jDvgA612q3VKX9Nr1VzdPKXaOMPyBBXgP1AB7rLxKaI9ZEPL8/UKipt5K75moroUfRvQDvNxgv+Cfc83gsH899AWyzXgMf5F8d6E993h34evWLRwxcLFS99d8YPbbURsWLLwrYVvv7XkrWU/+pzuyP8FyLeocnicY2BgYGQAgqtLZtwG0ddUuRNhNABJ9AW8AAB4nGNiYGBgYsAEAABwAAUA") format("woff");
```
Comment (sapegin) [inline src/styles/flat/font/social-likes.styl]: Please remove changes in this file, there are not related to your PR.

**Review by sapegin — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Comment by sapegin** [comment]
Almost, now there’s a whitespace change in that styl file ;-|

**Comment by RunnX** [comment]
@sapegin - I think the issue is now resolved.

**Review by sapegin — APPROVED** [review]

**Comment by sapegin** [comment]
Thanks, merged!

**Pull request merged**